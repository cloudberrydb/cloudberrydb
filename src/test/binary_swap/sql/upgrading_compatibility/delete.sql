DELETE FROM sales_heap WHERE product_id > 1000000;

DELETE FROM sales_ao WHERE product_id > 1000000;

DELETE FROM sales_aocs WHERE product_id > 1000000;

DELETE FROM sales_partition_heap WHERE product_id > 1000000;

DELETE FROM sales_partition_ao WHERE product_id > 1000000;

DELETE FROM sales_partition_aocs WHERE product_id > 1000000;

SELECT COUNT(*) FROM sales_heap;

SELECT COUNT(*) FROM sales_ao;

SELECT COUNT(*) FROM sales_aocs;

SELECT COUNT(*) FROM sales_partition_heap;

SELECT COUNT(*) FROM sales_partition_ao;

SELECT COUNT(*) FROM sales_partition_aocs;

