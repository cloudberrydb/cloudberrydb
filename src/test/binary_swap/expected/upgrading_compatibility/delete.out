DELETE FROM sales_heap WHERE product_id > 1000000;
DELETE FROM sales_ao WHERE product_id > 1000000;
DELETE FROM sales_aocs WHERE product_id > 1000000;
DELETE FROM sales_partition_heap WHERE product_id > 1000000;
DELETE FROM sales_partition_ao WHERE product_id > 1000000;
DELETE FROM sales_partition_aocs WHERE product_id > 1000000;
SELECT COUNT(*) FROM sales_heap;
 count 
-------
 10000
(1 row)

SELECT COUNT(*) FROM sales_ao;
 count 
-------
 10000
(1 row)

SELECT COUNT(*) FROM sales_aocs;
 count 
-------
 10000
(1 row)

SELECT COUNT(*) FROM sales_partition_heap;
 count 
-------
 10000
(1 row)

SELECT COUNT(*) FROM sales_partition_ao;
 count 
-------
 10000
(1 row)

SELECT COUNT(*) FROM sales_partition_aocs;
 count 
-------
 10000
(1 row)

